### DISCUSSION TITLE
Modula-2  unable to open /dev/tty: No such file or directory

### DISCUSSION THREAD

**Discussion by jonnygrant** [compiler-explorer/compiler-explorer#5829] [General]
https://godbolt.org/z/qMGK6xohc

unable to open /dev/tty: No such file or directory

Hello world! example not working for me. Any ideas?

**Comment by partouf** [comment]
I think this is mostly a bug in 13.1 and 13.2 in combination with our sandboxing.
This seems fixed in the current snapshot of gm2 https://godbolt.org/z/obEo67Kc3
